tooling-tests/src/test/resources/test-src/generated-sources/tool_test_data_type_1/lib_message_pb.bal

```diff
@@ -1,7 +1,13 @@
+import ballerina/protobuf;
+
+const string LIB_MESSAGE_DESC = "null";
```

Comment:
@MadhukaHarith92 why we have a null string here?

Reply:
Fixed in [cd09162](https://github.com/ballerina-platform/module-ballerina-grpc/pull/864/commits/cd091621678582c5ece93a252ba63c20f36f36e2)